public/data/sumitmukharjeeeeee.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Sumit Mukharjee",
+    "bio": "Learning and Exploring",
+    "avatar": "https://github.com/sumitmukharjeeeeee.png",
+    "links": [{
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/sumitmukharjeeeeee",
+        "icon": "github"
+    }, {
+        "name": "Follow me on Twitter",
+        "url": "http://twitter.com/SumitMukharjii",
+        "icon": "twitter"
+    }
+]
```

Comment:
Please can you format line 14, so the links closing square bracket is on line 13 @sumitmukharjeeeeee? I will check incase I am wrong though. Thanks